src/main/java/Parser.java

```diff
@@ -0,0 +1,80 @@
+                        }
+                    } else if (command.contains("deadline")) {
+                        try {
+                            if (command.equals("deadline ") || command.equals("deadline")) {
+                                throw new DukeException("    OOPS!!! The deadline cannot be empty.");
+                            } else {
+                                try {
+                                    xs.addDeadline(command);
+                                } catch (DateTimeParseException e) {
+                                    throw new DukeException("    OOPS!!! The date for the deadline should be written in the form: yyyy-mm-dd.");
+                                }
+                            }
+                        } catch (DukeException e) {
+                            System.out.println(e.getMessage());
+                        }
+                    } else if (command.contains("event")) {
+                        try {
+                            if (command.equals("event ") || command.equals("event")) {
+                                throw new DukeException("    OOPS!!! The description of an event cannot be empty.");
+                            } else {
+                                xs.addEvent(command);
+                            }
+                        } catch (DukeException e) {
+                            System.out.println(e.getMessage());
+                        }
+                    } else if (command.contains("todo")) {
+                        try {
+                            if (command.equals("todo ") || command.equals("todo")) {
+                                throw new DukeException();
+                            } else {
+                                xs.addTask(command);
+                            }
+                        } catch (DukeException e) {
+                            System.out.println(e.getMessage());
+                        }
+                    } else if (command.contains("delete")) {
+                        try {
+                            if (command.equals("delete ") || command.equals("delete")) {
+                                throw new DukeException("    OOPS!!! The task number cannot be empty.");
+                            } else {
+                                xs.delete(command);
+                            }
+                        } catch (DukeException e) {
+                            System.out.println(e.getMessage());
+                        }
+                    }  else {
+                        throw new DukeException("    OOPS!!! I'm sorry, but I don't know what that means :-(");
+                    }
+
+                } catch (DukeException e) {
+                    System.out.println(e.getMessage());
+                } finally {
+                    System.out.println("    ____________________________________________________________");
+                    return false;
+                }
```

Comment:
It's not my duty to review your coding standard since you're my second peer, but maybe you can consider of reading coding standard explanations on switch-case :)